app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchModel.kt

```diff
@@ -31,4 +33,8 @@ data class OverdueSearchModel(
   fun overdueSearchQueryChanged(searchQuery: String): OverdueSearchModel {
     return copy(searchQuery = searchQuery)
   }
+
+  fun loadStateChanged(overdueSearchProgressState: OverdueSearchProgressState): OverdueSearchModel {
```

Comment:
Same comment as above. Let's rename this to `progressStateChanged`

Reply:
Will update this